src/plugins/crypto.c

```diff
@@ -21,7 +21,7 @@
 #include <glib.h>
 #include <libcryptsetup.h>
 #include <nss.h>
-#include <libvolume_key.h>
+#include <volume_key/libvolume_key.h>
```

Comment:
This is covered by ``-I/usr/include/volume_key`` added to ``CFLAGS`` in *src/plugins/Makefile.am*. Please remove that from there.